.github/workflows/lock.yml

```diff
@@ -0,0 +1,20 @@
+name: Lock
+
+# yamllint disable-line rule:truthy
+on:
+  schedule:
+    - cron: "0 * * * *"
+
+jobs:
+  lock:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: dessant/lock-threads@v2.0.1
+        with:
+          github-token: ${{ github.token }}
+          issue-lock-inactive-days: "30"
+          issue-exclude-created-before: "2020-10-01T00:00:00Z"
+          issue-lock-reason: ""
```

Comment:
Are the blank reasons needed?
https://github.com/dessant/lock-threads-app/blob/master/src/lock.js#L46

Reply:
Yes, however, looking at the code, that is a bug... I think 🤔 
It should close without a reason in this case (instead of the `resolved` default).